x-pack/solutions/security/plugins/security_solution/public/common/hooks/constants.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export const URL_PARAM_KEY = {
+  appQuery: 'query',
+  /** @deprecated */
+  eventFlyout: 'eventFlyout', // TODO remove when we assume it's been long enough that all users should use the newer `flyout` key
```

Comment:
@PhilippeOberti,do we have telemetry?

Reply:
Not related to this PR I think but I can remove it if you want.